templates/edge-computing/index.html

```diff
@@ -10,468 +13,457 @@
-          <li class="p-list__item is-ticked">Reusing proven clouds primitive</li>
-          <li class="p-list__item is-ticked">Modular micro clouds, API integration with existing solutions</li>
-          <li class="p-list__item is-ticked">Fully automated, centrally managed, with low to no onsite operations</li>
-        </ul>
-      </div>
-    </div>
-  </section>
-
-  <hr class="p-rule u-hide--large u-hide--medium" />
-
-  <section class="p-strip is-bordered">
-    <div class="row">
-      <div class="col-6">
-        <div class="u-sv3 u-hide--small">
-          {{ image(url="https://assets.ubuntu.com/v1/8b16a766-Edge-industrial.svg",
-                    alt="",
-                    width="258",
-                    height="140",
-                    hi_def=True,
-                    loading="lazy") | safe
-          }}
-        </div>
-        <h2>Industry 4.0: inside the factory</h2>
-        <ul class="p-list">
-          <li class="p-list__item is-ticked">Scalable and low-latency HA cluster for critical industrial applications</li>
-          <li class="p-list__item is-ticked">Local data processing for more governance and less network costs</li>
-          <li class="p-list__item is-ticked">Compute power at the factory core for Al and digital twins applications</li>
-        </ul>
-        <p>
-          <a href="/internet-of-things/gateways">Connect IIoT devices to micro clouds&nbsp;&rsaquo;</a>
-        </p>
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>
+          Let’s discuss
+          <br />
+          your edge application
+        </h2>
       </div>
-      <div class="col-6">
-        <div class="u-sv3 u-hide--small">
-          {{ image(url="https://assets.ubuntu.com/v1/dfb83f4e-Edge-telecoms.svg",
-                    alt="",
-                    width="211",
-                    height="140",
-                    hi_def=True,
-                    loading="lazy") | safe
-          }}
-        </div>
-        <h2>Telco edge: 5G & edge computing</h2>
-        <ul class="p-list">
-          <li class="p-list__item is-ticked">Lightweight to run on small servers constrained by real estate</li>
-          <li class="p-list__item is-ticked">API-driven for providing on-demand compute and services</li>
-          <li class="p-list__item is-ticked">Distributed workloads, serving compute near the consumer</li>
-        </ul>
+      <div class="col">
         <p>
-          <a href="/telco">Read more on Telco@Edge&nbsp;&rsaquo;</a>
+          The Canonical team can deploy the first site with you to enable your edge strategy and upskill your teams. We provide long-term security maintenance and technical support for all the open source components you need to build your micro cloud.
         </p>
+        <hr class="p-rule--muted" />
+        <a href="/contact-us" class="js-invoke-modal p-button">Get in touch</a>
       </div>
```

Comment:
Please use `p-cta-block`

Reply:
I cannot, if I use `p-cta-block` that adds additional padding at the bottom, and Mattea asked to remove it.